installation-usage.md

```diff
@@ -453,6 +453,12 @@ test-recording-redis   Installed   15s
 Recording a SELinux profile would work the same, except you'd use `kind: SelinuxProfile`
 in the `ProfileRecording` object.
 
+Please note that log based recording does not have any effect if the recorded container
+is privileged, that is, the container's security context sets `privileged: true`. This
```

Comment:
Do we wanna behave in the same way if we already have a seccomp profile applied to the container?

Reply:
I think there might be a use-case for still recording a workload even if there's a profile applied to it. e.g. verifying that the applied profile is sufficient.

Reply:
But for the log enricher we would override the existing profile.

Reply:
For Seccomp? I thought it was only the case for SELinux.

Reply:
> Do we wanna behave in the same way if we already have a seccomp profile applied to the container?

Reply:
The webhook just updates the security context for log based seccomp profile recordings: https://github.com/kubernetes-sigs/security-profiles-operator/blob/1e4dfea95c075717e0fa9189380ce72f06da6a3b/internal/pkg/webhooks/recording/recording.go#L250-L268

Reply:
> > Do we wanna behave in the same way if we already have a seccomp profile applied to the container?
> 
> I'm sorry, can you elaborate? Are you thinking about the case where someone records a workload while already specifying the profile in the first place?

Yes, exactly. We could assume that users do not understand how it works in detail, so I'm wondering if we wanna make them aware that we overwrote their already used seccomp profile.

Reply:
OK, makes sense. btw it seems like we were overwriting the context for SELinux recording as well, so I added events to both.